test/integration/targets/nxos_gir/tests/common/sanity.yaml

```diff
@@ -0,0 +1,103 @@
+#---
+#- debug: msg="START TRANSPORT:{{ connection.transport }} nxos_gir sanity test"
+#
+#- name: "Setup"
```

Comment:
This file is fully commented out as the  module craters the device. Will uncomment when the module is fixed.

Reply:
@rahushen do you an issue opened for the module?

Reply:
@trishnaguha ... I had opened #27292 for this. I tried setting the connection timeout as you mentioned and it seems to work, but the module has other issues.